armada/handlers/k8s.py

```diff
@@ -58,3 +58,10 @@ def get_namespace_pod(self, namespace="default"):
         '''
 
         return self.client.list_namespaced_pod(namespace)
+
+    def get_all_pods(self):
+        '''
+        This will return a list of pods across all namespaces
+        '''
+
+        return self.client.list_pod_for_all_namespaces()
```

Comment:
this is to maintain the k8s actions in one file. The lib requires a lot of imports and body objects to execute the k8s api calls and dont want it to distract from the tiller actions in tiller.py this also allows us to switch libs possibly in the future without changing much in other files, just this one.